src/server/android/android.ts

```diff
@@ -150,20 +183,26 @@ export class AndroidDevice {
   }
 
   async close() {
-    const driver = await this._driver();
-    driver.close();
+    if (this._pollingWebViews)
```

Comment:
We should cleanup stuff like this on disconnect.

Reply:
Yeah, will follow up.